content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "Small steps towards sustainable and open research"
+
+[extra]
+
+author = "Aurélie Vancraeyenest, Samantha Wittke"
++++
+
+# Small steps towards sustainable and open research
+- or why you should care about Research Software Engineering
+
+A blog post summarizing the outcome of a discussion at Nordic-RSE conference May 2024 (https://nordic-rse.org/events/2024-in-person-conference/).
+
+#### What is a Research Software Engineer (RSE)?
+
+**The short answer** 
+People striving for "Better software, better research" 
+
+**The long answer**
+*Definition of Research Software ? (from Martin's slides)*
+Research software is software that one use to generate, process or analyse results that is intended to appear in a publication. It can be anything from a few lines of code, to a professionally developed software package.
+
+A Research Software Engineer is a person who combines an intricate understanding of research with professional software engineering expertise. RSE often have background in academia and have specialized in computing related areas, be it either software development, data analysis, (high-performance) computing, etc.
+
+Independently of their area of expertise, their common goal is better maintainability, sustainability of research software and better reproducibility of research outputs.
+
+**What an RSE is not**
+
+Even though they are in close collaboration with IT services, RSEs are not IT support people, as in IT manager or IT system specialists. They have a dual background (research/coding), they take part in research projects and speak the "same language" as researchers. 
+
+They also are not enforcers, they don't enforce technologies or methods but build on the tools already in use in research groups.  
```

Comment:
Disagree a bit on this. We (talking of my group) are actually "enforcers". Through collaboration with us we actually enforce use of git, some code style, etc. So while just question style advice is of course free from that, as soon as we collaborate with anyone we setup a github repo, change to some workable project setup etc. 

There might be a difference between RSE groups for larger institutions and small RSE groups like ours here, where we can serve at a more detailed level.

Reply:
Yes, we can soften. I assume what attendees meant here is that RSEs very much build on tools/tech researchers already know and use, but of course it shouldn't mean that RSEs don't also introduce best practice tools. I think Samantha's suggestion goes along the right direction.

Reply:
```suggestion
They may introduce best practice tools and techniques to a research group by showing the benefits and teaching them how to use them independently. But they can also estimate if a tool that the research group is used to is good enough for the job at hand.
```

Let's iteratively improve this sentence(s) :)

Reply:
After further thinking on this and re-reading through the workshop notes, I am wondering if it's not best to simply remove this sentence completely. It's not clear from the notes what the attendees meant here by "RSE are not enforcers" and by guessing or interpreting, we might end-up distorting what the initial message was. So I would suggest to just remove this sentence.

Reply:
True. Should we still keep something about "finding right tool for the job" ?

Reply:
Sure, let's then move this part a bit above. I'll add a suggestion based on yours.

Reply:
Good for me